linker/js/src/main/scala/org/scalajs/linker/irio/NodeVirtualIRFiles.scala

```diff
@@ -19,77 +19,245 @@ import scala.concurrent._
 import scala.scalajs.js
 import scala.scalajs.js.annotation.JSImport
 import scala.scalajs.js.typedarray._
+import scala.scalajs.js.typedarray.TypedArrayBufferOps._
 
-import java.io._
+import java.nio._
 
 import org.scalajs.ir
 
-class NodeVirtualScalaJSIRFile(val path: String, val relativePath: String) extends VirtualScalaJSIRFile {
-  val version: Option[String] =
-    NodeFS.statSync(path).mtime.map(_.getTime.toString).toOption
+trait NodeScalaJSIRContainer extends ScalaJSIRContainer {
```

Comment:
What is this trait useful for? It doesn't seem like anything uses it, or could use `ScalaJSIRContainer` instead.

(I'm only talking about the trait, not its companion object.)

Reply:
For consistency with the FileIRFile API: A `FileScalaJSIRContainer` exposes its underlying file (and we use this in the sbt plugin). So for Node I think we should be consistent.